projects/components/src/stepper/stepper-tab.component.ts

```diff
@@ -0,0 +1,27 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { AbstractControl } from '@angular/forms';
+import { IconType } from '@hypertrace/assets-library';
+
+import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '../content/content-holder';
+
+@Component({
+  selector: `ht-stepper-tab`,
+  template: CONTENT_HOLDER_TEMPLATE,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StepperTabComponent extends ContentHolder {
+  @Input()
+  public label!: string;
```

Comment:
Does it necessary to have this assertion here?

Reply:
Can also be kept optional.

Reply:
Updated.